src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,11 +1,30 @@
-export const TodoFilter = () => (
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Try to add `aria-label` attribute to buttons all over the project to avoid using `eslint-disable`.

Reply:
Great advice. Thanks))